src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -465,6 +465,117 @@ paths:
         connectionId: '${vpc_connection_id}'
         httpMethod: POST
 
+
+  #
+  # Anonymous Elector Document Applications
+  # --------------------------------------------------------------------------------
+  '/eros/{eroId}/anonymous-applications':
+    parameters:
+      - name: eroId
+        description: The ID of the Electoral Registration Office responsible for the application(s).
+        schema:
+          type: string
+        example: 'city-council'
+        in: path
+        required: true
+    options:
+      summary: CORS support
+      description: |
+        Enable CORS by returning correct headers
+      tags:
+        - Anonymous Elector Document Applications
+      responses:
+        200:
+          description: Default response for CORS method
+          headers:
+            Access-Control-Allow-Origin:
+              schema:
+                type: string
+            Access-Control-Allow-Methods:
+              schema:
+                type: string
+            Access-Control-Allow-Headers:
+              schema:
+                type: string
+          content: { }
+      x-amazon-apigateway-integration:
+        type: mock
+        requestTemplates:
+          application/json: |
+            {
+              "statusCode" : 200
+            }
+        responses:
+          default:
+            statusCode: "200"
+            responseParameters:
+              method.response.header.Access-Control-Allow-Headers: '''Content-Type,X-Amz-Date,Authorization,X-Api-Key'''
+              method.response.header.Access-Control-Allow-Methods: '''*'''
+              method.response.header.Access-Control-Allow-Origin: '''*'''
+            responseTemplates:
+              application/json: |
+                {}
+    get:
+      summary: Search for Anonymous Elector Document Applications
+      description: 'Search for Anonymous Elector Document Applications based on querying parameters'
+      tags:
+        - Anonymous Elector Document Applications
+      responses:
+        '200':
+          $ref: '#/components/responses/SearchApplications'
+      operationId: search-anonymous-elector-document-applications
+      parameters:
+        - name: pageSize
```

Comment:
On the basis there's a risk we may not include pagination in this release (I think we will, but there's a chance we won't), I'm not sure about including pagination yet as it suggests it's supported.

One option would be to make it clear it's not currently supported in the description and make sure the UI team know that, but I think it might be better/cleaner to exclude it for now (especially if we forget to update the description!).

Reply:
Ok I can remove pagination